modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ears.dm

```diff
@@ -5,18 +5,28 @@
 	relevent_layers = list(BODY_BEHIND_LAYER, BODY_ADJ_LAYER, BODY_FRONT_LAYER)
 	genetic = TRUE
 
-/datum/sprite_accessory/ears/is_hidden(mob/living/carbon/human/human, obj/item/bodypart/bodypart)
-	if(
-		(human.head && (human.head.flags_inv & HIDEHAIR)) \
-		|| (human.wear_mask && (human.wear_mask.flags_inv & HIDEHAIR)) \
-		|| human.try_hide_mutant_parts \
-		|| !bodypart \
-	)
-		//This line basically checks if we FORCE accessory-ears to show, for items with earholes like Balaclavas and Luchador masks
-		if(!(human.head && (human.head.flags_inv & SHOWSPRITEEARS) || (human.wear_mask && (human.wear_mask.flags_inv & SHOWSPRITEEARS)) || !bodypart))
-			return TRUE
-	return FALSE
-
+/datum/sprite_accessory/ears/is_hidden(mob/living/carbon/human/wearer, obj/item/bodypart/bodypart)
+	if(!wearer.head || !bodypart)
+		return FALSE
+//	Can hide if wearing hat
+	if(key in wearer.try_hide_mutant_parts)
+		return TRUE
+//	Exception for MODs
+	if(istype(wearer.wear_suit, /obj/item/clothing/suit/mod))
+		return FALSE
+//	This line basically checks if we FORCE accessory-ears to show, for items with earholes like Balaclavas and Luchador masks
+	else if(!(wearer.head \
+			&& (wearer.head.flags_inv & SHOWSPRITEEARS) \
+				|| (wearer.wear_mask \
+				&& (wearer.wear_mask.flags_inv & SHOWSPRITEEARS)) \
+				)
+			)
+		return TRUE
+//	Hide accessory if flagged to do so
+	else if(wearer.head.flags_inv & HIDEHAIR \
+			|| (wearer.wear_mask && (wearer.wear_mask.flags_inv & HIDEHAIR)) \
+		)
+		return TRUE
```

Comment:
These two are meant to be combined together, otherwise headwear will no longer show ears. `SHOWSPRITEEARS` is an override to `HIDEHAIR` so that it still shows ears even if the hair is hidden.

Reply:
fixed, sorry about that...